
The 3.8 Kernel headers for if_bridge.h uses the in6_addr struct
which is defined by netinet/in.h which is not included prior.

Upstream-Status: Pending

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: git/src/tethering.c
===================================================================
--- git.orig/src/tethering.c
+++ git/src/tethering.c
@@ -35,6 +35,7 @@
 #include <string.h>
 #include <fcntl.h>
 #include <linux/if_tun.h>
+#include <netinet/in.h>
 #include <linux/if_bridge.h>
 
 #include "connman.h"
